src/assets/fonts/fontello/css/mantle.css

```diff
@@ -0,0 +1,247 @@
+@font-face {
```

Comment:
I fixed fonts in raster foundry by correcting the paths (IE remove `?52812087` from the font files) in the font css file and importing it in  whatever module uses the fonts.

`require('../../../asseets/fonts/fontello/css/mantlecss')`

url loader should take care of the imports in the sass file